PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+from pymoab import core, types, tag
+from sys import argv
+
+mb = core.Core()
+
+#Read the data file with the graveyard to be removed.
+mb.load_file(argv[1])
```

Comment:
Maybe add a check before loading the file that the user actually supplied an argument. See the "Check Input Arguments" here for info about defensive programming. :) https://kkiesling.github.io/python-novice-gapminder-custom/32b-defensive-prog/